mer_verify_kernel_config

```diff
@@ -223,7 +223,8 @@ CONFIG_CGROUP_PERF		y,!	# systemd (optional): http://0pointer.de/blog/projects/c
 CONFIG_CGROUP_SCHED		y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html
 CONFIG_BLK_CGROUP		y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html
 CONFIG_NET_CLS_CGROUP		y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html
-CONFIG_NETPRIO_CGROUP		y,!	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html
+CONFIG_NETPRIO_CGROUP		y,!,<3.15	# systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html, only valid if kernel version is <3.15
+CONFIG_CGROUP_NET_PRIO          y,!>=3.15     # systemd (optional): http://0pointer.de/blog/projects/cgroups-vs-cgroups.html, only valid if kernel version is <3.15
```

Comment:
Indentation mismatch, first line has tabs like all other lines, latter line has spaces.

Reply:
Not all have one but I replaced my spaces with a tab.